wolftpm/tpm2_types.h

```diff
@@ -208,10 +212,24 @@ typedef int64_t  INT64;
 
 #ifndef WOLFTPM_CUSTOM_TYPES
     #include <stdlib.h>
+
+    #ifndef XHTONS
+        /* WOLFCRYPT_ONLY means no wolfio and no arpa/inet.h */
+        #ifdef WOLFCRYPT_ONLY
+            #define XHTONS(s) ((((s) & 0xff) << 8) | (((s) & 0xff00) >> 8))
+        #else
+            #include <arpa/inet.h>
+            #define XHTONS(s)         htons((s))
+        #endif
+    #endif
+
     #define XSTRTOL(s,e,b)    strtol((s),(e),(b))
     #define XATOI(s)          atoi((s))
+
 #endif
 
+#define RSA_SIG_SZ 256
```

Comment:
Maybe just use `MAX_RSA_KEY_BYTES` in the example instead of adding a new macro?

Reply:
Can this be removed now?